src/handlers/payout/action.ts

```diff
@@ -75,7 +77,7 @@ export const handleIssueClosed = async () => {
   const comment = `### [ **[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]** ](${payoutUrl})\n` + "```" + shortenRecipient + "```";
   const comments = await getAllIssueComments(issue.number);
   const commentContents = comments.map((i) => i.body);
-  const exist = commentContents.find((content) => content.includes(comment));
+  const exist = commentContents.find((content) => content.includes(TEMPLATE_PAYOUT_URL));
```

Comment:
It should also check that the comment's author is the `ubiquibot` to eliminate cases when permit URL is mentioned in the comments for whatever reason